### DISCUSSION TITLE
Design Planning

### DISCUSSION THREAD

**Discussion by egdiala** [NenLabs/nen-ui#8] [Team Posts]
Whats up guys? I already have some components we can start designing but there's a structured workflow I want us to have. I was thinking we can have a call for this so I can explain better. @NenUI/design

**Comment by onis77** [comment]
Let's have a call

**Comment by egdiala** [comment]
What day and time would be perfect? @NenUI/design

**Comment by egdiala** [comment]
@NenUI/design 
I created videos that explains our workflow on loom.
This is the [first video](https://www.loom.com/share/8cb939fdbe36425e881a08a144b7e49f) and this is the [second video](https://www.loom.com/share/d46b27484d4c49e3ab1d99ac2a2636db).
